vizro-core/examples/dev/app.py

```diff
@@ -513,6 +715,148 @@
     ],
 )
 
+# LAYOUT ------------------------------------------------------------------
+
+grid_layout = vm.Page(
+    title="Grid layout",
+    layout=vm.Grid(grid=[[0, 0, 0, 0], [1, 1, 3, 3], [1, 1, 4, 4], [2, 2, 5, 5], [2, 2, 6, 6]]),
+    components=[
+        vm.Card(
+            text="""
+                ### Card #1
+
+                Lorem ipsum dolor sit amet, consetetur sadipscing no sea elitr sed diam nonumy.
+                Sed diam voluptua. At vero eos et accusam et justo no duo dolores et ea rebum.
+            """
+        ),
+        vm.Card(
+            text="""
+                ### Card #2
+
+                Sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat.
+                Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+            """
+        ),
+        vm.Card(
+            text="""
+                ### Card #3
+
+                Sed diam voluptua. At vero eos et accusam et justo no duo dolores et ea rebum.
+                Lorem ipsum dolor sit amet, consetetur sadipscing no sea est elitr dolor sit amet.
+            """
+        ),
+        vm.Card(
+            text="""
+                ### Card #4
+
+                Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+                Sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat.
+            """
+        ),
+        vm.Card(
+            text="""
+                ### Card #5
+
+                Lorem ipsum dolor sit amet, consetetur sadipscing no sea est elitr dolor sit amet.
+                Sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat.
+            """
+        ),
+        vm.Card(
+            text="""
+                ### Card #6
+
+                Sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat.
+                Lorem ipsum dolor sit amet, consetetur sadipscing no sea est elitr dolor sit amet.
+            """
+        ),
+        vm.Card(
+            text="""
+                ### Card #7
+
+                Lorem ipsum dolor sit amet, consetetur sadipscing no sea elitr sed diam nonumy.
+                Sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat.
+            """
+        ),
+    ],
+    description="""
+        Use Grid when you have a specific layout in mind—like a dashboard with clearly defined sections
+        (e.g. top summary row, bottom detail view).
+    """,
+)
+
+flex_layout = vm.Page(
+    title="Flex layout",
+    layout=vm.Flex(
+        direction="column",
+        wrap=True,
+        gap="40px",
+    ),
```

Comment:
Can we do direction row and wrap = True. It should look like this example visually? https://vizro.readthedocs.io/en/stable/pages/user-guides/custom-figures/#dynamic-number-of-cards
You might have to specifiy a height and width to the cards then.

```suggestion
    layout=vm.Flex(
        direction="row",
        wrap=True,
        gap="40px",
    ),
```